docs/.vuepress/components/ApiSlot.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <div>
-    <p>Find more information about using Slots in the article, <cdr-link :href="$withBase('/getting-started/as-a-developer/?active-link=add-content-via-slots')">Getting Started as a Developer</cdr-link></p>
-    <div class="api-slot" v-for="(apiSlot, index) in this.$slots.default" :key="apiSlot.text">
+    <p v-show="slotsGettingStartedLink">Find more information about using Slots in the article, <cdr-link :href="$withBase('/getting-started/as-a-developer/?active-link=add-content-via-slots')">Getting Started as a Developer</cdr-link></p>
```

Comment:
This could just be v-if since we aren't going to be show/hiding it.

Reply:
Fixed.